packages/graphiql/src/new-components/themes/default/index.js

```diff
@@ -0,0 +1,37 @@
+import Layout from './Layout';
+
+const palette = {
+  neutral: {
+    20: '#939393',
+    90: `rgba(0, 0, 0, 0.1)`,
+    100: '#fff',
+  },
+  pink: {
```

Comment:
i think @orta wanted to call this fuscia? oooh but how will we officially pronounce fuscia? sounds like a standards body decision

Reply:
let's